### PR TITLE
Hero

### PR DESCRIPTION
**Pull request by Sashikbear**
I added html to section hero

### CODE REVIEW

File: blocks/hero/hero.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.hero {
+  background-color: #2f70fe;
```
Comment (resdenia) [inline blocks/hero/hero.css:2]: Lets add root variables for main colors

File: pages/index.css [hunk 2]
```diff
@@ -3,4 +3,5 @@
 @import url(../blocks/page/page.css);
 @import url(../blocks/header/header.css);
 @import url(../blocks/main/main.css);
+@import url(../blocks/hero/hero.css);
```
Comment (resdenia) [inline pages/index.css:6]: Amazing!!

**Review by resdenia — CHANGES_REQUESTED** [review]
It's AMAZING! THATNK YOU FOR YOUR WORK! Please  check my comments and we ready to merge.

**Review by nitzanye — APPROVED** [review]
Great job Alex!!!

**Pull request merged**